docs/sources/static/set-up/install/install-agent-linux.md

```diff
@@ -48,35 +83,86 @@ You can install Grafana Agent in static mode on Linux.
    sslcacert=/etc/pki/tls/certs/ca-bundle.crt
    ```
 
-1. Verify that the repository is properly configured using `yum-config-manager`:
+1. Optional. Verify the Grafana repository configuration:
```

Comment:
I can't remember, but weren't we supposed to use the form "(Optional)" for optional steps?

Reply:
This was a point that was raised recently within the Style Council. The Google Style guide says `Optional:` https://developers.google.com/style/procedures#optional-steps  Looking again, there wasn't a final decision to use `(Optional)` or `Optional:`  The style guide still says to use brackets. I'll follow up on this.